### DISCUSSION TITLE
Airgapped - Missing Zeek container on Forward Node

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#5830] [General] [answered]
Hi,

We're missing Zeek on all our Forward nodes and the only solution I've found is to run the following on the master: 
sudo so-docker-refresh

When I tried to run this on an Airgapped installed of security onion it tired to connect to the internet.

Any ideas on who I can get Zeek installed on the forward node in an airgapped environment?

**Comment by dougburks** [comment, marked as answer]
When you ran Setup, did you choose to use Zeek for metadata instead of Suricata?

Is this a new installation or an existing installation that you updated?

If existing, did you ever have Zeek on the forward nodes?

**Reply by wildlyunder** [reply]
Hi,,

I used Suricata for metadata.

This is an existing installation.

I thought Zeek was on the forward nodes... I only noticed recently it wasn't working.

**Reply by dougburks** [reply]
If you chose Suricata instead of Zeek for metadata, then there is no reason for Zeek to run.

**Reply by wildlyunder** [reply]
If we wanted to use Zeek and Suricata how would we enable both?

**Reply by dougburks** [reply]
You can use Suricata for Alerts and Zeek for metadata. We do not support running both Suricata metadata and Zeek metadata on the same box as this would create duplicate metadata.

**Reply by wildlyunder** [reply]
How do I change to use Suricata for alerts and zeek for metadata? Would this impact our use of ET Pro?

**Reply by dougburks** [reply]
> How do I change to use Suricata for alerts and zeek for metadata?

The process for switching from Zeek metadata to Suricata metadata is at https://docs.securityonion.net/en/2.3/suricata.html#metadata and you should be able to perform similar steps but set `mdengine` to `ZEEK`.

> Would this impact our use of ET Pro?

ET PRO is a NIDS ruleset and really only applies to Suricata alerting not metadata.
